backend/config/settings/base.py

```diff
@@ -146,11 +146,18 @@
 CORS_ORIGIN_WHITELIST = env.list("CORS_ORIGIN_WHITELIST", default="http://localhost:3000")
 ALLOWED_HOSTS = env.list("DJANGO_ALLOWED_HOSTS", default=["localhost"])
 
+# URLS
+BASE_URL = env("DJANGO_BASE_URL", default="https://www.indokntnu.no")
```

Comment:
Consider changing `BASE_URL` to `FRONTEND_BASE_URL` too as it is a bit unclear what url this actually is.. What is the "base"?

Reply:
Great suggestion, fixed in afe6e3f